gci/students.py

```diff
@@ -1,8 +1,7 @@
-import re
 import logging
+import re
```

Comment:
Reason for this change?

Reply:
It was a result of auto-formatting by pyCharm IDE. Will undo that change